<launch>

 <!-- joy node -->
  <node respawn="true" pkg="joy"
        type="joy_node" name="md49_joy" >
    <param name="dev" type="string" value="/dev/input/js0" />
    <param name="deadzone" value="0.12" />
  </node>

 <!-- Axes -->
  <param name="axis_linear" value="1" type="int"/>
  <param name="scale_linear" value="1" type="double"/>
  <param name="axis_angular" value="0" type="int"/>
  <param name="scale_angular" value="3.14" type="double"/>

  <param name="axis_head_pan" value="3" type="int"/>
  <param name="scale_head_pan" value="1.40" type="double"/>
  <param name="axis_head_tilt" value="4" type="int"/>
  <param name="scale_head_tilt" value="1.40" type="double"/>

  <node pkg="joy_md49" type="joy_md49_teleop" name="teleop"/>
</launch>
